src/statscollector/DefaultStatsCollector.ts

```diff
@@ -414,11 +414,7 @@ export default class DefaultStatsCollector implements StatsCollector {
   }
 
   isValidRawMetricReport(rawMetricReport: RawMetricReport): boolean {
-    return (
-      (this.isValidChromeRawMetric(rawMetricReport) ||
-        this.isValidStandardRawMetric(rawMetricReport)) &&
-      this.isValidSsrc(rawMetricReport)
-    );
+    return this.isValidStandardRawMetric(rawMetricReport) && this.isValidSsrc(rawMetricReport);
```

Comment:
Your PR doesnt say it - but have you tested against all the browsres and ensure they are correctly populated? Could you also update the PR on the testing steps?

Reply:
Tested in Chrome, Firefox and Safari.
Updated the testing steps in description,

Reply:
Awesome! Assuming mobile is covered as well. Could i request you to link this PR to the https://github.com/aws/amazon-chime-sdk-js/issues/1964 issue in the description so it will provide ongoing update of the changes?

Reply:
>Awesome! Assuming mobile is covered as well. Could i request you to link this PR to the https://github.com/aws/amazon-chime-sdk-js/issues/1964 issue in the description so it will provide ongoing update of the changes?

Just found that the above message already linked the PR to #issue 1964 😄
![image](https://user-images.githubusercontent.com/20471438/157299642-680c4b07-14eb-4ef9-9e32-46455038e946.png)